### DISCUSSION TITLE
What is the watermark on the navigation for?

### DISCUSSION THREAD

**Discussion by Humb3l** [pimcore/pimcore#11117] [Q&A / Support] [answered]
I recently installed the newest Pimcore X Skeleton, and I noticed some weird watermark at the navigation, that is coming from the following css class: _x-title-text-default:after_
And I wonder what is this for? And how do I remove it.

Mfg Humbel

**Comment by NiklasBr** [comment, marked as answer]
Unsure of its purpose, but it will disappear if you clear your browser's cache and reload the admin page.
